client/src/components/Info.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  background-color: var(--white);
+  height: 80vh;
```

Comment:
there's not really a need to set the height, if you use padding and margins to your elements!